docs/highlights/2021-06-02-fix-vector-metric-naming.md

```diff
@@ -0,0 +1,22 @@
+---
+last_modified_on: "2021-06-02"
+$schema: ".schema.json"
+title: "Telemetry units for duration metrics fixed"
+description: "A few telemetry metric names incorrectly had `nanoseconds_total` in their name when they are actually `seconds`"
+author_github: "https://github.com/jszwedko"
+pr_numbers: [7373]
+release: "0.14.0"
+hide_on_release_notes: false
+tags: ["type: announcement", "domain: metrics"]
```

Comment:
```suggestion
tags: ["type: breaking change", "domain: metrics"]
```

Reply:
Aha, thanks, I was wondering how that labeling happened.